components/side-drawer/src/vwc-side-drawer-base.ts

```diff
@@ -93,50 +108,49 @@ export class VWCSideDrawerBase extends LitElement {
 		const topBar = this.hasTopBar ? this.renderTopBar() : '';
 		const scrim = (this.type === 'modal' && this.open) ? this.renderScrim() : '';
 		const alternate = this.alternate ? 'vvd-scheme-alternate' : undefined;
+		const end = this.position === 'end';
 
 		const classes = {
-			'side-drawer--alternate': this.alternate,
-			'side-drawer--dismissible': dismissible,
-			'side-drawer--modal': modal,
-			'side-drawer--open': this.open,
+			'side-drawer-alternate': this.alternate,
+			'side-drawer-dismissible': dismissible,
+			'side-drawer-modal': modal,
+			'side-drawer-open': this.open,
+			'side-drawer-end': end,
+		};
+
+		const contentClasses = {
```

Comment:
Why twice?

Reply:
Cause the content should get only 'side-drawer-end' class and not the others

Reply:
use sibling selector